tests/Tests/ORM/Mapping/ClassMetadataTest.php

```diff
@@ -870,13 +870,23 @@ public function testAssociationOverrideKeepsDeclaringClass(): void
     {
         $cm = new ClassMetadata(Directory::class);
         $cm->mapManyToOne(['fieldName' => 'parentDirectory', 'targetEntity' => Directory::class, 'cascade' => ['remove'], 'declared' => Directory::class]);
-        $cm->setAssociationOverride('parentDirectory', ['cascade' => '']);
+        $cm->setAssociationOverride('parentDirectory', ['cascade' => ['remove']]);
```

Comment:
Why this change in the test?

Reply:
Previously `cascade` was ignored, now it's leading to an error as it's supposed to be an array of strings. I reckon this test was originally done to pass something and see if nothing (declaring class) breaks